src/test/java/org/mitre/synthea/export/FHIRDSTU2ExporterTest.java

```diff
@@ -239,12 +241,9 @@ public void testObservationAttachment() throws Exception {
     int age = 35;
     long birthTime = time - Utilities.convertTime("years", age);
     person.attributes.put(Person.BIRTHDATE, birthTime);
-
-    Payer.loadNoInsurance();
-    for (int i = 0; i < age; i++) {
-      long yearTime = time - Utilities.convertTime("years", i);
-      person.coverage.setPayerAtTime(yearTime, Payer.noInsurance);
-    }
+    PayerManager.loadPayers(new Location("Massachusetts", null));
```

Comment:
Ideally, you do not want to specify a location in the unit tests like "Massachusetts" because it will break synthea-international support. Instead, you want to do something like

```
String testStateDefault = Config.get("test_state.default", "Massachusetts");
PayerManager.loadPayers(new Location(testStateDefault, null));
```